src/components/TransactionItemComponent/DataCeils/MerchantCell.tsx

```diff
@@ -0,0 +1,21 @@
+import useThemeStyles from '@hooks/useThemeStyles';
+import TextWithTooltip from '@components/TextWithTooltip';
+import React from 'react';
+import colors from '@styles/theme/colors';
+import type DataCellProps from './DateCellProps';
+
+function MerchantCell({transactionItem, showTooltip, isLargeScreenWidth}: DataCellProps) {
+    const styles = useThemeStyles();
+
+    const merchantToDisplay = !transactionItem?.merchant || transactionItem?.merchant === "(none)" ? '' : transactionItem.merchant;
```

Comment:
can we move `(none)` to a const? can you verify that there is no const with this value already existing?